package.json

```diff
@@ -53,6 +61,20 @@
 				"mac": "cmd+alt+up",
 				"linux": "shift+alt+up",
 				"when": "editorTextFocus"
+			},
+			{
+				"command": "smartmulticursor.lastCursorLeft",
+				"key": "ctrl+alt+left",
+				"mac": "cmd+alt+left",
+				"linux": "shift+alt+left",
+				"when": "editorTextFocus"
+			},
+			{
+				"command": "smartmulticursor.lastCursorRight",
+				"key": "ctrl+alt+right",
+				"mac": "cmd+alt+right",
+				"linux": "shift+alt+right",
+				"when": "editorTextFocus"
```

Comment:
I don't think it is good to override "Move Editor into Next/Previous Group". Better remove it and let the user rewrite the keybinding if desired.

Reply:
No prob at all. Will remove.

Reply:
Fixed in https://github.com/idanpa/vscode-smartmulticursor/pull/8/commits/51c4b75d76de036382f2ebd8037b00877006e529